pkg/cli/image/info/info.go

```diff
@@ -78,6 +80,9 @@ func NewInfo(streams genericclioptions.IOStreams) *cobra.Command {
 	o.SecurityOptions.Bind(flags)
 	flags.StringVarP(&o.Output, "output", "o", o.Output, "Print the image in an alternative format: json")
 	flags.StringVar(&o.FileDir, "dir", o.FileDir, "The directory on disk that file:// images will be read from.")
+	flags.BoolVar(&o.ClusterICSP, "cluster-icsp", o.ClusterICSP, "When set to true, look for alternative image sources from ImageContentSourcePolicy objects in cluster, honor the ordering of those sources, and fail if an ImageContentSourcePolicy is not found in cluster.")
```

Comment:
I don't like that this fails if there is no ICSP.  As a user, if I want ICSP to be used, I'll specify the flag in scripts and tolerate when a cluster doesn't have an ICSP.  The flag should be `use-cluster-icsp`.  There's no real disadvantage for a user using an ICSP, or there not being one on some clusters.   If we had an explicit ICSP locator flag, then I would agree we would fail.

Reply:
Roger that.

Reply:
Although we do fail when you specify a file with `--icsp-file` and we have issues reading the file.